cmd/edv-rest/startcmd/start.go

```diff
@@ -89,6 +89,7 @@ func startEDV(parameters *edvParameters) error {
 
 	handlers := edvService.GetOperations()
 	router := mux.NewRouter()
+	router.UseEncodedPath()
```

Comment:
(this is what allowed the router to properly handle url-encoded strings in the path)